Resources/Locale/en-US/tabletop/tabletop.ftl

```diff
@@ -1,5 +1,10 @@
 ﻿## TabletopGameComponent
 tabletop-verb-play-game = Play Game
+tabletop-default-board-name = Generic Board
```

Comment:
What about "Board game"?

Reply:
you're supposed to never see this, but sure